function-reference/query-functions/text.xml

```diff
@@ -63,6 +73,14 @@ query variables and similar state.
       As <c>use_enable</c>, but <c>--with-</c> or <c>--without-</c>.
     </ti>
   </tr>
+  <tr>
+    <ti>
+    <c>in_iuse flag</c>
+    </ti>
+    <ti>
+      Returns true if <c>flag</c> is in IUSE_EFFECTIVE, false otherwise.
```

Comment:
I'd rather avoid mentioning IUSE_EFFECTIVE because it is only a conceptual variable in PMS, i.e. it isn't exported to the ebuild environment (and therefore the devmanual doesn't explain it).

Maybe say something like this: "... if the ebuild can use &lt;c&gt;flag&lt;/c&gt; in &lt;c&gt;use&lt;/c&gt; queries, ..."